### DISCUSSION TITLE
Welcome to camplayer Discussions!

### DISCUSSION THREAD

**Discussion by raspicamplayer** [raspicamplayer/camplayer#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by michaelsage** [comment]
Hi, I have a pi zero with a 2.8 inch waveshare GPIO based screen. I have tried running the demo, but only appear to get console output text. Has anyone had this running on a GPIO based (rather than HDMI based) screen?

**Comment by kenwiesner** [comment]
Just found this project and it's great! I struggled a little but was able to get it working with my Unifi Protect on my UDM Pro. The stream URLs were way to complicated to retype so I manually built the config and with the exception of one of my cameras which had a highres screen resolution that it didn't like, it works perfectly! Next step is to try audio.

**Comment by HeliSteve** [comment]
I just found this project and I love it. Nice and easy to set up. I am wondering if the Camplayer 2 was ever released? I would like to be able to control I/O pins on the camera modules.
